cachi2/core/models/input.py

```diff
@@ -216,6 +216,7 @@ class RpmPackageInput(_PackageInputBase):
     """Accepted input for a rpm package."""
 
     type: Literal["rpm"]
+    add_rpm_summary: bool = False
```

Comment:
nitpick - it's part of `RpmPackageInput` already so the 'rpm' part of the option name is redundant, but at the same time the name itself doesn't indicate this is an SBOM only flag - what about `sbom_inlude_summary` OR `include_summary_in_sbom`? What I'm after is some kind of naming pattern that could be reasonably reusable for other such flags we might be required to add, IOW `include_XYZ_in_sbom`.

Reply:
Good point, done.